src/utils/logging.jl

```diff
@@ -196,10 +214,15 @@ ensure that all events get flushed.
 - `file_mode::String=w+`: mode used when opening log file
 - `tracker::Union{LogEventTracker, Nothing}=LogEventTracker()`: optionally track log events
 - `set_global::Bool=true`: set the created logger as the global logger
+- `suppression_period::Int`: duration in seconds in which a log event will be supressed if
+   the count exceeds `max_count`.
 
 # Example
 ```Julia
 logger = configure_logging(filename="mylog.txt")
+@info "hello world"
+@info "hello world" max_count = 5
+@info "hello world" max_count = 5 suppression_period = 10
```

Comment:
This should be `_max_count` and `_suppression_period` to avoid collisions with local variables.